tests/test_performance.py

```
@@ -0,0 +1,397 @@
+from app.main import create_app
+from app.schemas.astrology import AstrologicalSubjectRequest
+
+
+class LoadTester:
+    """Load testing utility for the API."""
+
+    def __init__(self, base_url: str = "http://localhost:8000"):
+        self.base_url = base_url
+        self.client = httpx.AsyncClient(base_url=base_url, timeout=30.0)
+
+    async def test_single_request(self, endpoint: str, method: str = "GET", data: Dict = None) -> Dict[str, Any]:
```

Ruff (0.13.1)

PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)